### PR TITLE
Fix loadash vulnerability - Closes #4656

### LINKED ISSUE #4656: Fix lodash vulnerability
### Expected behavior
Update the lodash version to https://github.com/lodash/lodash/pull/4336
### Actual behavior
https://github.com/lodash/lodash/pull/4336
### Steps to reproduce
`npm audit` against `release/3.0.0`
### Which version(s) does this affect? (Environment, OS, etc...)
3.0.0

### PR DESCRIPTION
**Pull request by ManuGowda**
### What was the problem?
https://github.com/lodash/lodash/pull/4336
### How did I solve it?
Updating the lodash version 
### How to manually test it?
`npm audit`
### Review checklist

- [ ] The PR resolves #4656 
- [ ] All new code is covered with unit tests
- [ ] Relevant integration / functional tests are added
- [ ] Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
- [ ] Documentation has been added/updated

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -5892,9 +5892,9 @@
 			}
 		},
 		"lodash": {
-			"version": "4.17.11",
-			"resolved": "https://registry.npmjs.org/lodash/-/lodash-4.17.11.tgz",
-			"integrity": "sha512-cQKh8igo5QUhZ7lg38DYWAxMvjSAKG0A8wGSVimP07SIUEK2UO+arSRKbRZWtelMtN5V0Hkwh5ryOto/SshYIg==",
+			"version": "4.17.15",
+			"resolved": "https://registry.npmjs.org/lodash/-/lodash-4.17.15.tgz",
+			"integrity": "sha512-8xOcRHvCjnocdS5cpwXQXVzmmh5e5+saE2QGoeQmbKmRS6J3VQppPOIt0MnmE+4xlZoumy0GPG0D0MVIQbNA1A==",
```
Comment (nazarhussain) [inline package-lock.json]: Why we still keep the old version in the package.json file? 

https://github.com/LiskHQ/lisk-sdk/blob/a898540c67b5e601020c964889a5d64c3b54b64a/framework/package.json#L73

Comment (nazarhussain) [reply package-lock.json]: There are other libraries which are depending on `^4.17.11` version. Can we make sure all those dependence are also updated.

Comment (ManuGowda) [reply package-lock.json]: apart from `framework` others are a transient dependency. so if you run `npm audit` in the root it won't complain. so should be good. for lisk-core I will update them as well.

**Comment by ManuGowda** [comment]
> @ManuGowda There some left out reference of the old version still.
> 
> https://github.com/LiskHQ/lisk-sdk/blob/b74f9081bdc609b321c52091976ea9419ee05a27/framework/package-lock.json#L7188

That's the required version of the dependency of a dependency.. it explicitly requiring `4.17.11`, I am not sure you can update them. Do know how?

**Comment by nazarhussain** [comment]
@ManuGowda If `4.17.11` of lodash have vulnerability that we have to upgrade, then it must be upgraded from everywhere in the project.

**Comment by nazarhussain** [comment]
Its in devDepedencies so all good with the code.

**Pull request merged**